src/main/java/com/instana/operator/customresource/InstanaAgentSpec.java

```diff
@@ -237,6 +246,20 @@ public String getClusterName() {
     return clusterName;
   }
 
+  public String getAgentImagePullPolicy() { return agentImagePullPolicy; }
+
+  public Boolean getAgentOtelActive() { return agentOtelActive; }
+
+  public void setAgentOtelActive(Boolean agentOtelActive) { this.agentOtelActive = agentOtelActive; }
+
+  public Integer getAgentOtelPort() { return agentOtelPort; }
+
+  public void setAgentOtelPort(Integer agentOtelPort) {
+    this.agentOtelPort = agentOtelPort;
+  }
+
+  public void setAgentImagePullPolicy(String imagePullPolicy) { this.agentImagePullPolicy = imagePullPolicy; }
```

Comment:
Would be nice to relocate this next to the `getAgentImagePullPolicy`.

Reply:
Ok